libs/traits/src/fee.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright 2023 Centrifuge Foundation (centrifuge.io).
+
+// Centrifuge is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version (see http://www.gnu.org/licenses).
+
+// Centrifuge is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+/// Trait to add fees to a pool
+pub trait AddPoolFees {
+	type PoolId;
+	type FeeBucket;
```

Comment:
Not really, suppose I can remove that type.

Reply:
Ah, the reason I didn't do that was that this require adding `cfg-types` as dep to `cfg-traits` which we don't want to avoid circular deps (types depends on traits already).

Reply:
And would it make sense to move `FeeBucket` to `cfg-traits`?

From my perspective, it's okay to have types in `cfg-traits` if those types exist as part of the interface the trait offers.

Reply:
Done! Thanks for the recommendation!